plugin/vim/quick-lint-js.vim/lua/lspconfig/quick_lint_js.lua

```diff
@@ -0,0 +1,30 @@
+local configs = require("lspconfig/configs")
+local util = require("lspconfig/util")
+
+configs.quick_lint_js = {
+    default_config = {
+        cmd = {"quick-lint-js", "--lsp-server"},
+        filetypes = {"javascript"},
+        root_dir = function(fname)
+            return util.find_node_modules_ancestor(fname) or
+                       util.find_git_ancestor(fname) or
```

Comment:
Why are we searching for Git repositories? Git seems unrelated to quick-lint-js or JavaScript projects. Is this logic standard and recommended practice for Neovim plugins?

Reply:
Those are ways to choose the ``root_dir`` which will spawn a server instance. Since most configs in lspconfig are like that, I just kept it the same way.